main.go

```diff
@@ -3,14 +3,19 @@ package main
 import (
 	"github.com/cyverse-de/de-stats/api"
 	"github.com/cyverse-de/echo-middleware/redoc"
-
+	"github.com/cyverse-de/de-stats/cron"
 	"github.com/labstack/echo"
 	"github.com/labstack/echo/middleware"
 
 	_ "github.com/cyverse-de/de-stats/docs"
 )
 
 func main() {
+	db := cron.InitDB()
+	defer db.Close()
+	amount := 10
+	days := 100
+	cron.GetTopApps(db, amount, days)
```

Comment:
I'm still curious about this code. Is it just lingering test code?

Reply:
yes, thought i took that out